### PR TITLE
fix: search

### PR DESCRIPTION
**Pull request by sarahschwartz**

### CODE REVIEW

File: .github/workflows/docs-scrapper.yml [hunk 1]
```diff
@@ -19,7 +19,6 @@ jobs:
```
Comment (luizstacio) [inline .github/workflows/docs-scrapper.yml:19]: Do we need this CI yet?

**Review by luizstacio — CHANGES_REQUESTED** [review]

**Review by pedronauck — CHANGES_REQUESTED** [review]
I guess the search results style needs some fixes

CleanShot 2023-07-11 at 12 41 42@2x

**Comment by sarahschwartz** [comment]
> I guess the search results style needs some fixes
> 
> CleanShot 2023-07-11 at 12 41 42@2x

yes, any keywords that are in the book titles right now look like this. Do you think we should just hide lvl0 results that are just matching with the book name? I noticed in our other books the h1 titles also show up like this right now too.

**Comment by pedronauck** [comment]
@sarahschwartz checking other documentation like the on in NextJS, when searching they are showing the name of the page and also content in a nested style like that:
CleanShot 2023-07-14 at 15 31 34@2x

this is very good to create this sense of navigation and search in the same time, but of course, this will need some level of customization right now... I guess that for the moment, we can go with just hiding the lv0 indeed

**Review by pedronauck — DISMISSED** [review]

**Review by luizstacio — APPROVED** [review]

**Pull request merged**